Documentation/Metrics/arangodb_replication2_replicated_log_append_entries_rtt.yaml

```diff
@@ -0,0 +1,17 @@
+name: arangodb_replication2_replicated_log_append_entries_rtt
+introducedIn: "3.9.0"
+help: |
+  Histogram of the RTT of appendEntries requests in us.
```

Comment:
I found the "in us" slightly misleading when reading it. I only later realized that "us" refers to microseconds here. All ambiguity would go away by replacing "us" with "microseconds" here.